aiohttp/connector.py

```diff
@@ -813,10 +813,13 @@ def _get_fingerprint(self, req):
         fingerprint = self._get_fingerprint(req)
 
         try:
-            hosts = await self._resolve_host(
+            # Cancelling this lookup should not cancel the underlying lookup
+            #  or else the cancel event will get broadcast to all the waiters
+            #  across all connections.
+            hosts = await  asyncio.shield(self._resolve_host(
```

Comment:
Is there two spaces between `await` and `asyncio.shield()`?

Reply:
fixed